devel/gearmand/Makefile

```diff
@@ -1,7 +1,7 @@
 # $NetBSD: Makefile,v 1.50 2020/06/02 08:23:36 adam Exp $
 
-DISTNAME=	gearmand-1.1.18
-PKGREVISION=	13
+DISTNAME=	gearmand-1.1.19
+PKGREVISION=	1
```

Comment:
When updating package `PKGREVISION` should be removed.

Reply:
Thanks, removed.